ts/client/src/client.ts

```diff
@@ -1601,24 +1601,42 @@ export class MangoClient {
       clientOrderId,
       limit,
     );
+
     const settleIx = await this.serum3SettleFundsIx(
       group,
       mangoAccount,
       externalMarketPk,
     );
 
-    return await this.sendAndConfirmTransactionForGroup(group, [
-      ...placeOrderIxes,
-      settleIx,
-    ]);
+    const ixs = [...placeOrderIxes, settleIx];
+
+    if (closeOpenOrdersAccount) {
```

Comment:
is this still a useful flag, given that we saw that when you actually take you'll not be able to close without processing some events? (at least that's what I heard)

Reply:
no, removed now